app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Katello
+  class Api::V2::UpstreamSubscriptionsController < Api::V2::ApiController
+    before_action :find_organization
+
+    resource_description do
+      description "Subscriptions management on the CDN."
```

Comment:
CDN refers to cdn.redhat.com which serves actual content, but upstream here is actually the Red Hat Subscription Management platform - so I think we should be more precise